ATeam-FantasyGame/Location.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+class Location
+{
+private:
+	std::string name; // Name of the Location
+	std::string description; // A description of the Location
+
+public:
+	Location(std::string name, std::string description) : name(name), description(description) {} // Location Constructor
```

Comment:
Interesting, certainly a valid way to define your constructor (maybe even preferred by some), but isn't this duplicative to the constructor defined in your Location.cpp file?